src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -85,8 +148,84 @@ const formConfig = {
             properties: {}
           }
         },
-        // pageTwo: {},
-        // pageThree: {},
+        evidenceType: {
+          title: (formData, { pagePerItemIndex }) => formData.disabilities[pagePerItemIndex].diagnosticText,
```

Comment:
You might want to use `get` for this, since the array won't always be there.

Reply:
Oh right. Good point.